File: 3300-minimum-element-after-replacement-with-digit-sum/3300-minimum-element-after-replacement-with-digit-sum.java
class Solution {
    public int minElement(int[] nums) {
        int mini=Integer.MAX_VALUE;
        for(int i=0; i<nums.length; i++){
            int val=0;
            while(nums[i]>0){
                val+=(nums[i]%10);
                nums[i]/=10;
            }
            if(val<mini)mini=val;
        }
        return mini;
    }
}

File: 3300-minimum-element-after-replacement-with-digit-sum/3300-minimum-element-after-replacement-with-digit-sum.py
class Solution:
    def minElement(self, nums: List[int]) -> int:
        mini=max(nums)
        for i in nums:
            val=0
            while i>0:
                val+=(i%10)
                i=int(i/10)
            if val<mini:
                mini=val
        return mini